app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,19 +29,43 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * 저장된 모든 상품의 집합을 반환합니다.
```

Comment:
수학적으로 `List`는 `Ordered Set`이기 때문에 틀린 말이 아닙니다. 그런데 Java에서 집합이라면 `Set` 인터페이스를 먼저 떠올리게 하죠. 이렇게 수정하는 편이 함께 작업하는 동료들을 덜 혼란스럽게 할 거에요.
```suggestion
     * 저장된 모든 상품의 목록을 반환합니다.
```

Reply:
전혀 생각치 못한 부분이었네요 .. 
협업을 할 때는 이런 디테일한 부분을 신경 쓰는 게 중요할 것 같습니다.